bot/client.js

```diff
@@ -57,9 +57,16 @@ client.on('message', (message) => {
   }
 });
 
-// If User joins server this message will be sent
+// Event listener for new members
 client.on('guildMemberAdd', (member) => {
-  member.sendMessage(`${member.user.username}, Welcome to the Full Sail Armada! These are the terms of service...`);
+  // Send welcome message to members who join
+  member.sendMessage(`${member}, Welcome to the Full Sail Armada! These are the terms of service...`);
```

Comment:
Also, "sendMessage" is being deprecated, so it is recommended to change those methods to just "send".